contrib/scalar/t/t9099-scalar.sh

```diff
@@ -154,4 +154,13 @@ test_expect_success '`scalar clone` with GVFS-enabled server' '
 	)
 '
 
+test_expect_success 'scalar delete without enlistment shows a usage' '
+	test_expect_code 129 scalar delete
+'
+
+test_expect_success 'scalar delete with enlistment' '
+	scalar delete cloned &&
```

Comment:
BTW I _think_ the test failures are due to my mishandling a non-running FSMonitor daemon in `stop_fsmonitor_daemon()`...